llvm/test/CodeGen/AMDGPU/GlobalISel/atomic_optimizations_mul_one.ll

```diff
@@ -1,5 +1,7 @@
+; XFAIL: *
+; XFAIL: *
```

Comment:
Why 3 XFAILs?

Reply:
because of the stupid script which I used to update the LIT files :)
They are switched off to be able to run the PSDB and look at what is broken.